module/behaviour/skills/HeadBehaviourSoccer/src/HeadBehaviourSoccer.cpp

```diff
@@ -62,66 +54,30 @@ namespace module::behaviour::skills {
     using utility::support::Expression;
 
     inline Eigen::Vector2d screenAngularFromObjectDirection(const Eigen::Vector3d& v) {
+        // Returns {yaw, pitch}
```

Comment:
This would be better as doc directives, info here https://nubook.nubots.net/guides/general/code-conventions#documentation
same for any other functions